tasks/main.yml

```diff
@@ -215,16 +264,48 @@
       name: consul
       state: started
       enabled: yes
+    when: ansible_os_family != "Windows"
+
+  - name: Install NSSM
+    win_chocolatey:
+      name: nssm
+      state: latest
```

Comment:
@judy @brianshumate Shouldn't there be a when clause here checking for os_family == "Windows"?
(Didn't test)